### PR TITLE
Inventory: Added `IItem` and `ICarriableItem` supported Inventory

### PR DESCRIPTION
**Pull request by Alf21**

### CODE REVIEW

File: code/items/equipments/TTTEquipment.cs
Comment (mzegar) [inline code/items/equipments/TTTEquipment.cs:16]: Can this just be removed?

**Review by mzegar — APPROVED** [review]
Personally, LGTM, if you want to wait for another review please go ahead.

File: code/items/equipments/TTTEquipment.cs
Comment (Alf21) [reply code/items/equipments/TTTEquipment.cs:16]: Kept it because it worked and afaik gonna work in the future again (still opened issue)

**Pull request merged**
